cmd/ci-secret-bootstrap/main.go

```diff
@@ -277,30 +262,21 @@ func constructDockerConfigJSON(client secrets.ReadOnlyClient, dockerConfigJSONDa
 	for _, data := range dockerConfigJSONData {
 		authData := secretbootstrap.DockerAuth{}
 
-		registryURL := data.RegistryURL
-		if registryURL == "" {
```

Comment:
Why is the registry url resolving logic being removed?

Reply:
First we had only the registry-url field that it was supposed to be a field in bitwarden, but we added the `registryURL` that gets the value directly from the config. We kept the old one to keep backwards compatibility and now it is not being used at all. So it is safe to remove it as well.